Makefile

```diff
@@ -37,7 +51,7 @@ double-conversion.a:			\
 		fixed-dtoa.o		\
 		string-to-double.o	\
 		strtod.o
-	$(AR) rcsD $@ $^
```

Comment:
Could you just change this to `rcs` and leave out the platform check?

Reply:
easy enough!